python/lsst/ts/genericcamera/genericcameracsc.py

```diff
@@ -1005,6 +1000,11 @@ async def configure(self, config):
                 f"No config found for sal_index={self.salinfo.index}"
             )
 
+        if config.s3instance != "none":
+            self.use_lfa = True
```

Comment:
I think there might be a silent bug here. 

If you first configure the CSC with `config.s3instance != "none"` then re-configure it with `config.s3instance != "none"`, the second time `self.use_lfa` won't be set, and will continue to be `True`.

Consider doing:

```
self.use_lfa = config.s3instance != "none"
```

instead.